src/restapi/fHelp.ts

```diff
@@ -8,9 +8,11 @@ import {CommandTrigger} from '../utils/constants';
 import {isUserSystemAdmin} from '../utils';
 
 export async function fHelp(req: Request, res: Response): Promise<void> {
-    let helpText = getHeader();
-    helpText += getCommands(req.body);
-    helpText += getPostText();
+    const helpText = [
+        getHeader(),
+        getCommands(req.body),
+        getPostText(),
+    ].join('\n');
```

Comment:
Apparently, you were not joining these string with line breaks. Not sure if this change is intended.

Reply:
Nice catch.  The line breaks are not needed here.